pkg/console/errors/trusted_ca_errors.go

```diff
@@ -0,0 +1,22 @@
+package errors
+
```

Comment:
Not sure if this is being used.

Reply:
no, its from the first drwaft, will remove it